scripts/src/tests/services/users.spec.ts

```diff
@@ -213,26 +213,26 @@ describe("api tests for v2 users", async () => {
 		expect(res.payload.test).toEqual(payload.test);
 	});
 
-	test("simulate http deprecation error", async () => {
-		const myApp = await helpers.createApp(generateId(IdPrefix.App));
-		const user = await helpers.createUser({ appId: myApp.id });
-		const token = (await AuthToken.findOne({ userId: user.id }))!;
-
-		let res = await mock(app)
-			.get(`/v2/users/me`)
-			.set("Authorization", `Bearer ${ token.id }`)
-			.set("x-request-id", "123");
-		expect(res.status).toBe(200);
-
-		res = await mock(app)
-			.get(`/v2/users/me`)
-			.set("Authorization", `Bearer ${ token.id }`)
-			.set("x-request-id", "123")
-			.set("x-simulate-deprecation-error", "true");
-
-		expect(res.status).toBe(410);
-		expect(res.body.code).toBe(4101);
-	});
+	// test("simulate http deprecation error", async () => {
+	// 	const myApp = await helpers.createApp(generateId(IdPrefix.App));
+	// 	const user = await helpers.createUser({ appId: myApp.id });
+	// 	const token = (await AuthToken.findOne({ userId: user.id }))!;
+
+	// 	let res = await mock(app)
+	// 		.get(`/v2/users/me`)
+	// 		.set("Authorization", `Bearer ${ token.id }`)
+	// 		.set("x-request-id", "123");
+	// 	expect(res.status).toBe(200);
+
+	// 	res = await mock(app)
+	// 		.get(`/v2/users/me`)
+	// 		.set("Authorization", `Bearer ${ token.id }`)
+	// 		.set("x-request-id", "123")
+	// 		.set("x-simulate-deprecation-error", "true");
+
+	// 	expect(res.status).toBe(410);
+	// 	expect(res.body.code).toBe(4101);
+	// });
```

Comment:
Instead of removing this test, you can change it to test your new x-kin-blockchain-version header

Reply:
Fixed the test, but `helpers.createApp` hardcoded to kin2
We will add it in future PRs?